HISTORY.md

```diff
@@ -1,30 +0,0 @@
-# History
```

Comment:
Please leave the history alone. In fact, if you're making a change, you should update it.